code/controllers/subsystem/ticker.dm

```diff
@@ -606,6 +606,10 @@ SUBSYSTEM_DEF(ticker)
 	fdel(F)
 	WRITE_FILE(F, the_mode)
 
+/// Returns if either the master mode or the forced secret ruleset matches the mode name.
+/datum/controller/subsystem/ticker/proc/is_mode(mode_name)
+	return GLOB.master_mode == mode_name || GLOB.secret_force_mode == mode_name
```

Comment:
```suggestion
#define IS_MODE(mode_name) (GLOB.master_mode == mode_name || GLOB.secret_force_mode == mode_name)
```
I assume this proc won't be redefined (which is a good thing)

Reply:
I'm personally fine forgoing this. We might as well use what little semblance of namespacing we have.